File: scaffoldUI/scaffold-project/src/app/Models/invoice.ts
export interface Invoice {
    invoiceId: number;
    invoiceNumber: number;
    invoiceCurrencyId: number;
    vendorId: number;
    invoiceAmount: number;
    invoiceReceivedDate: Date;
    invoiceDueDate: Date;
    isActive: boolean;
  }
  

File: vms/vms/vms/Models/Invoice.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace vms.Models
{
    [Table("Invoices")]
    public class Invoice
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int InvoiceId { get; set; }
        public Int64 InvoiceNumber { get; set; }
        public int InvoiceCurrencyId { get; set; }
        public int VendorId { get; set; }
        public decimal InvoiceAmount { get; set; }
        public DateTime InvoiceReceivedDate { get; set; }
        public DateTime InvoiceDueDate { get; set; }
        public bool IsActive { get; set; }
    }
}
